Copy component docs (#1503)
hashicorp/design-system: website/docs/components/copy/snippet/partials/accessibility/accessibility.md

## Conformance rating

<Doc::Badge @type="warning">Conditionally Conformant</Doc::Badge>

> When the default variant of this component is used, there should not be any WCAG conformance issues with this component.

Reviewer: ```suggestion
When the default variant of the component is used, there should not be any WCAG conformance issues.
```

> However, the truncation feature is not conformant, because a keyboard-only user has no method of accessing truncated content. Users should be aware of this when evaluating whether or not to include truncation in your copy snippet.

Reviewer: ```suggestion
However, the truncation feature is not conformant, as a keyboard-only user has no method of accessing truncated content. Users should be aware of this when evaluating whether or not to include truncation in their copy snippet.
```

Reviewer 2: Grammarly suggestion + we should talk to consumers, so I suggest:

```suggestion
However, the truncation feature is not conformant because a keyboard-only user cannot access the truncated content. Please be aware of this when evaluating whether or not to include truncation in your copy snippet.
```
